content/fr/docs/setup/custom-cloud/kubespray.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Installer Kubernetes avec Kubespray (on-premises et fournisseurs de cloud)
```

Comment:
s/on-premises/sur site/ d'aprés le GDoc

Reply:
I thought that `on-premises` is more understandable than `sur site` for the title. Even in France we always use the word `on-premises`. Waiting for other opinions to take a decision.

Reply:
Il faut au moins rester cohérent. Je préfère on-premises parce que ça a l'avantage d'être unique ce qui facilite les recherche. Tandis que `site` peut être utilisé dans plein d'autres contextes.

Reply:
👍 il faut mettre le GDoc à jour alors, je préfère aussi on-prem

Reply:
done

Reply:
"on-premise (sur site)" ?